sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -211,10 +241,11 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     checkExistence(sql("describe functioN abcadf"), true,
       "Function: abcadf not found.")
 
-    checkExistence(sql("describe functioN  `~`"), true,
-      "Function: ~",
-      "Class: org.apache.hadoop.hive.ql.udf.UDFOPBitNot",
-      "Usage: ~ n - Bitwise not")
+    // TODO: Re-enable this test after we fix SPARK-14335.
```

Comment:
The output of this test shows that we are loading a hive function. But, we actually have implementation of `~`. With this PR, we will lazily load hive's built-in functions and `describe` command will not trigger the load. So, this test will fail with a message like `Undefined function`. SPARK-14335 will fix this problem.